images/base/scripts/common/setup.sh

```diff
@@ -83,7 +83,12 @@ fi
 
 # First install protoc
 cd /tmp
-git clone https://github.com/google/protobuf.git
+if [ x$MACHINE = xs390x ]
+then
+    git clone https://github.com/gongsu832/protobuf.git
```

Comment:
Same comment at https://github.com/hyperledger/fabric/pull/2086#issuecomment-231217871

This clones a custom protobuf library with zero explanation. Is there a better way to pull only whatever custom fix is needed?